yaml/pois.yaml

```diff
@@ -926,45 +952,46 @@ filters:
           - [ 15,   1 ]
         default: { clamp: { min: 12, max: 14, value: { sum: [ { col: zoom }, 1.066 ] } } }
     output:
-      - *output_properties
-      - {kind: waterfall, height: {col: height}}
+      <<: *output_properties
+      kind: waterfall,
```

Comment:
Similar to @matkoniecz's [comment](https://github.com/tilezen/vector-datasource/pull/1343#issuecomment-318299590), there's a trailing comma here.